### PR TITLE
Feature/elg 69/add fiche buttons

### PR DESCRIPTION
**Pull request by dannybetts**

### CODE REVIEW

File: src/main/resources/templates/common.ftl [hunk 1]
```diff
@@ -13,6 +13,10 @@
   <@govukButton.button buttonText="Download arrow image" buttonClass="govuk-button"/>
 </#macro>
 
+<#macro packageCalculatorDownloadButtons>
+  <@govukButton.buttonWithSecondaryAction primaryButtonText="Download arrow image" secondaryButtonText="Download fiche"/>
```
Comment (matteason) [inline src/main/resources/templates/common.ftl]: "Download label", not "Download arrow image"

File: src/main/resources/templates/common.ftl [hunk 2]
```diff
@@ -41,7 +45,7 @@
 
 <#-- Template for standard product forms.
 Includes the wrapping form element, the generate label button and optionally the supplier name and model fields -->
-<#macro standardProductForm title includeSupplierNameModel=true includeRescaledInternetLabellingFields=false showInsetText=true beforeStandardInsetText="">
+<#macro standardProductForm title includeSupplierNameModel=true includeRescaledInternetLabellingFields=false showInsetText=true beforeStandardInsetText="" packageCalculatorForm=false>
```
Comment (matteason) [inline src/main/resources/templates/common.ftl:48]: Maybe call the param `isPackageCalculatorForm` to make it slightly clearer?

File: src/main/resources/templates/govuk/button.ftl [hunk 3]
```diff
@@ -9,3 +9,8 @@
     </button>
   </#if>
 </#macro>
+
+<#macro buttonWithSecondaryAction primaryButtonText secondaryButtonText primaryButtonClass="" primaryButtonUrl="#" secondaryButtonClass="govuk-button--tertiary" secondaryButtonUrl="#" buttonsDisabled=false>
```
Comment (matteason) [inline src/main/resources/templates/govuk/button.ftl:13]: Wrap buttons in `<div class="govuk-button-group">` per https://design-system.service.gov.uk/components/button/secondary-combo/index.html (it spaces them out a bit more)

File: src/main/resources/templates/govuk/button.ftl [hunk 4]
```diff
@@ -9,3 +9,8 @@
...
+    <@button buttonText=primaryButtonText buttonClass=primaryButtonClass buttonUrl=primaryButtonUrl disabled=buttonsDisabled/>
```
Comment (matteason) [inline src/main/resources/templates/govuk/button.ftl:14]: Nit, 4 space indent instead of 2?

**Review by matteason — CHANGES_REQUESTED** [review]

**Review by FernandoRamirez-De-Aguilar — APPROVED** [review]

**Pull request merged**